mixer/test/client/rbac_permissive_policy/rbac_permissive_policy_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2018 Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client_test
+
+import (
+	"fmt"
+	"testing"
+
+	"istio.io/istio/mixer/test/client/env"
+)
+
+// Report attributes from a GET request for Rbac permissive mode at policy level.
+const reportAttributes = `
+{
+  "connection.mtls": false,
+  "context.protocol": "http",
+  "context.proxy_error_code": "*",
+  "mesh1.ip": "*",
+  "mesh2.ip": "*",
+  "mesh3.ip": "*",
+  "rbac.permissive.effective_policy_id": "details-reviews-viewer",
+  "rbac.permissive.response_code": "200",
+  "request.headers": {
+      ":method": "GET",
+      ":path": "/echo",
+      ":authority": "*",
+      "x-forwarded-proto": "http",
+      "x-request-id": "*",
+      "user-agent": "Go-http-client/1.1",
+      "content-length": "*",
+      "accept-encoding":"gzip"
+  },
+  "request.host": "*",
+  "request.method": "GET",
+  "request.path": "/echo",
+  "request.scheme": "http",
+  "request.size": 0,
+  "request.time": "*",
+  "request.total_size": 0,
+  "request.url_path": "/echo",
+  "request.useragent": "Go-http-client/1.1",
+  "response.code": 403,
+  "response.duration": "*",
+  "response.headers": {
+      "date": "*",
+      "content-length": "*",
+      ":status": "403",
+      "server": "envoy",
+      "content-type":"text/plain"
+  },
+  "response.size": "*",
+  "response.time": "*",
+  "response.total_size": "*",
+  "source.namespace": "XYZ11",
+  "source.uid": "POD11",
+  "target.name": "target-name",
+  "target.namespace": "XYZ222",
+  "target.uid": "POD222",
+  "target.user": "target-user"
+}
+`
+
+const rbacPolicyPermissiveFilter = `
+- name: envoy.filters.http.rbac
+  config:
+    rules:
+      policies: {}
+    shadow_rules:
+      policies:
+        details-reviews-viewer:
+          permissions:
+          - and_rules:
+              rules:
+              - or_rules:
+                  rules:
+                  - header:
+                      name: ":method"
+                      exact_match: GET
+          principals:
+          - any: true
+`
+
+// Test senario that setting permissive mode on policy level.
+// Response code is 403 since Rbac is turned on and denied by default.
```

Comment:
Rbac is turned on and denied by default => authorization is turned on and is deny-by-default.